etc/init.d/mptcpd.in

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env /lib/init/init-d-script
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (c) 2022, Intel Corporation
+
+### BEGIN INIT INFO
+# Provides:          mptcpd
+# Required-Start:    $syslog $network $remote_fs
+# Required-Stop:     $syslog $network $remote_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: run mptcpd
+# Description:       Debian init script to start the Multipath TCP daemon.
+### END INIT INFO
+
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:@libdir@
+export LD_LIBRARY_PATH
```

Comment:
Setting this environment variable like this probably isn't good idea when supporting multi-arch.